core/feconf.py

```diff
@@ -22,11 +22,18 @@
 import datetime
 import enum
 import os
+import sys
 
 from core.constants import constants
 
 from typing import Dict, List, Union
-from typing_extensions import TypedDict
+
+# We require typing_extensions which is present in thiry_party so we add it to
+# path here.
```

Comment:
Is this really only needed for `typing_extensions`?

Reply:
From what I have seen yes. Before it never broke becuase the import was happening inside python_utils which is called rather early so it was always in path.